File: SSH/sshandroid/app/src/main/java/es/tipolisto/sshconnect/Models/ClienteComando.java
package es.tipolisto.sshconnect.Models;

public class ClienteComando {
    private int id;
    private int idCliente;
    private int idComando;
    private String nombre;

    public ClienteComando(int id, int idCliente, int idComando, String nombre) {
        this.id = id;
        this.idCliente = idCliente;
        this.idComando = idComando;
        this.nombre = nombre;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdCliente() {
        return idCliente;
    }

    public void setIdCliente(int idCliente) {
        this.idCliente = idCliente;
    }

    public int getIdComando() {
        return idComando;
    }

    public void setIdComando(int idComando) {
        this.idComando = idComando;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    @Override
    public String toString() {
        return "ClienteComando{" +
                "id=" + id +
                ", idCliente=" + idCliente +
                ", idComando=" + idComando +
                '}';
    }
}

File: SSH/ssh/modelos/ClienteComando.php
<?php
class ClienteComando{
    private $id;
	private $idCliente;
    private $idComando;
    function __construct($id, $idCliente, $idComando){
		$this->id=$id;
        $this->idCliente=$idCliente;
        $this->idComando=$idComando;
    }
    		
	function getId(){
		return $this->id;
    }
    		
	function getIdCliente(){
		return $this->idCliente;
    }
    		
	function getIdComando(){
		return $this->idComando;
	}
    function toString(){
		return "Cliente-comando: id: ".$this->id.", idCliente: ".$this->idCliente.", idComando: ".$this->idComando;	
	}
}

?>